modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cache/ClientCacheRequest.java

```diff
@@ -69,6 +82,15 @@ protected boolean isKeepBinary() {
         return (flags & FLAG_KEEP_BINARY) == FLAG_KEEP_BINARY;
     }
 
+    /**
+     *  Gets a value indicating whether expiry policy is set in this request.
+     *
+     * @return expiry policy flag value.
+     */
+    protected boolean withExpiryPolicy() {
```

Comment:
This method is not used outside of this class. I would inline it in the constructor, and use `expiryPolicy != null` check in `rawCache` to simplify.

Reply:
Thanks, converted to private. 
I think the explicit method declaration is a kind of self-documented approach, I would prefer to keep it as is. Thoughts?

Reply:
No objections